packages/account/src/Components/forms/idv-form.tsx

```diff
@@ -78,14 +73,9 @@ const IDVForm = ({
                     example_format,
                 };
             });
-
-            if (can_skip_document_verification) {
```

Comment:
why are we removing this skip check, is it not required anymore?

Reply:
Accordingly to the scope of the card the option for skipping IDV will be not only for real account creation flow, there is just a difference for the text of that option. so we don’t need this check anymore